.woodpecker.yml

```diff
@@ -1,19 +1,27 @@
 steps:
+  parse:
+    image: golang
+    environment:
+      TZ: Europe/Berlin
+    commands:
+      - cd calendar_parse
+      - go run .
+    when:
+      event: [push, tag, deployment, pull_request, cron]
+      branch: [staging, main]
```

Comment:
Not sure if it is a good idea to render the calendar via a cron job on the server. This can lead to long delays when publishing or changing events. Sometimes it might be neccessary to do short term changes.

Reply:
Maybe we can run it more often?